pkg/crc/errors/multierror.go

```diff
@@ -8,6 +8,14 @@ import (
 	"github.com/code-ready/crc/pkg/crc/logging"
 )
 
+type vmNotExist string
+
+func (v vmNotExist) Error() string {
+	return string(v)
+}
+
+const VMNotExist vmNotExist = "Machine crc does not exist. Use 'crc start' to create it"
```

Comment:
Maybe remove `crc` and just say `Machine does not exist. Use 'crc start' to create it`

Reply:
@guillaumerose Yes that what I was thinking, good that we have same thought here.